src/atoms/cc-button.js

```diff
@@ -462,6 +468,52 @@ export class CcButton extends LitElement {
           width: var(--width);
         }
 
+        /* circle waiting mode - keyframes */
+        @keyframes rotate {
+          100% {
+            transform: rotate(360deg);
+          }
+        }
+
+        @keyframes stretch {
+          0% {
+            stroke-dasharray: 1, 200;
+            stroke-dashoffset: 0;
+          }
+          50% {
+            stroke-dasharray: 90, 200;
+            stroke-dashoffset: -35px;
+          }
+          100% {
+            stroke-dashoffset: -124px;
+          }
+        }
+
+        /* circle waiting mode - partial opacity */
+        button.circle[waiting] {
+          opacity: 1;
+        }
+        button.circle[waiting] .text-wrapper img {
+          opacity: 0.25;
+        }
+
+        /* circle waiting mode - core animation */
+        .circle-loader {
+          --bcw-speed: 2s;
```

Comment:
It stands for "Button Circle Waiting"... :see_no_evil: 
I dropped a kinda-random prefix to "couple" the custom property (which should be private) with the component but I'm not really sure if it's worth it since it's not really understandable.